ppr-ui/src/interfaces/mhr-registration-interfaces/MhrUnitNoteIF.ts

```diff
@@ -0,0 +1,28 @@
+import { UnitNoteDocTypes, UnitNoteStatusTypes } from '@/enums'
+import { AddressIF, SubmittingPartyIF } from '..'
+
+export interface MhrRegistrationUnitNoteIF {
+  documentType: UnitNoteDocTypes
+  documentId: string
+  documentRegistrationNumber: string
+  documentDescription: string
+  createDateTime: string
+  effectiveDateTime: string
+  expiryDateTime: string
+  status: UnitNoteStatusTypes
+  remarks: string
+  givingNoticeParty: SubmittingPartyIF
+  destroyed: boolean
+}
+
+export interface MhrUnitNoteIF {
+  clientReferenceId: string
+  attentionReference: string
+  submittingParty: {
+    businessName: string
+    address: AddressIF
+    phoneNumber: string
+    emailAddress: string
+  }
+  note: MhrRegistrationUnitNoteIF
+}
```

Comment:
The unit notes are really part of the registration process, stand alone filing per se. 

The latter IF would work for the filing itself though.